geth/jail/internal/loop/loop_test.go

```diff
@@ -0,0 +1,72 @@
+package loop
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	gomock "github.com/golang/mock/gomock"
+
+	"github.com/status-im/status-go/geth/jail/internal/vm"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestLoopSuite(t *testing.T) {
+	suite.Run(t, new(LoopSuite))
+}
+
+type LoopSuite struct {
+	suite.Suite
+
+	loop   *Loop
+	cancel context.CancelFunc
+
+	mockTask     *MockTask
+	loopMockCtrl *gomock.Controller
+}
+
+func (s *LoopSuite) SetupTest() {
+	s.loopMockCtrl = gomock.NewController(s.T())
+	s.mockTask = NewMockTask(s.loopMockCtrl)
```

Comment:
There is really no need to bring this `gomock` dependency here. We have this a lot in other tests, so probably that was a reasoning behind using it here, but that's mostly legacy tests and need to be cleaned up. `Task` is an interface and it's extremely easy to add dummy implementation here:

```go
type DummyTask struct{}

func (*DummyTask) SetID(int64)                 {}
func (*DummyTask) GetID() int64                { return 1 }
func (*DummyTask) Cancel()                     {}
func (*DummyTask) Execute(*vm.VM, *Loop) error { return nil }
```
and use it in tests:
```go
task := &DummyTask{}
err := s.loop.Add(task)
...
err = s.loop.Ready(task)
```

It will result in absolutely the same test coverage, but will be much more clear, explicit and simple.